indra/llprimitive/llgltfmaterial.h

```diff
@@ -120,12 +130,6 @@ class LLGLTFMaterial : public LLRefCount
     //       heightmaps cannot currently be described as finite enclosed
     //       volumes.
     // See also LLPanelRegionTerrainInfo::validateMaterials
-    bool mDoubleSided = false;
-
-
-    // These fields are local to viewer and are a part of local bitmap support
-    typedef std::map<LLUUID, LLUUID> local_tex_map_t;
-    local_tex_map_t                  mTrackingIdToLocalTexture;
```

Comment:
@AndrewMeadows changed the order of the fields to save some bits. That's relevant to resolving this merge conflict. You may be interested in choosing a location for mEmissiveStrength to optimize the packing (although the savings may only be relevant server-side)

Reply:
Yeah - I wasn't sure what the specific ordering was supposed to be here.  I'll look at the git-blame and reorder as needed.

Reply:
I restored the packing from the previous merge - I'm generally assuming there's a very good reason for it.

Reply:
Grokking this further, it appears there's a comment/warning further down that `LLGLTFMaterial::getHash` uses `mLocalTexDataDigest` to determine the hash offset of the material object. (see "IMPORTANT: do not move this member down") Any fields prior to `mLocalTexDataDigest` will not get included in the hash. I didn't consider this before, but this probably takes precedence over @AndrewMeadows ' size optimizations.

This also would explain why the merge broke. There was a fix to GLTF hashing (and therefore render batching) in a maintenance branch.